### PR TITLE
BULK VALIDATION various fixes

### PR DESCRIPTION
**Pull request by mSarts**
## Category

- [x] Content fix
- [ ] New article

## Related issues

no issue number - part of the Validation clean-up efforts.

## What's in this Pull Request?

This pull request fixes 44 issues across 23 files.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[1ed9277](https://github.com/mSarts/sp-dev-docs/commits/1ed9277b8919cd2aeb19d423a80284e1860fb7b1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/attachment-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/attachment-element-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/attachment-element-deploymentmanifest?branch=pr-en-us-8052) | [Details](#user-content-16aae9269872ae047255b5dfff7b8f96-1879d565)
[docs/schema/attachments-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/attachments-element-deploymentmanifest.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/attachments-element-deploymentmanifest?branch=pr-en-us-8052) | [Details](#user-content-e2ed7cc00fce0c30c3462cfc51610d5b-1879d565)
[docs/schema/authenticationdata-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationdata-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/authenticationdata-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-72ae00230304cbcd5a01b48a755c455f-1879d565)
[docs/schema/authenticationdata-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationdata-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/authenticationdata-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-43fd1ca35e429d8b28c571e58cf5c91d-1879d565)
[docs/schema/authenticationinformation-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationinformation-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/authenticationinformation-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-75903820bea060aa52b8ce7cf3f6a326-1879d565)
[docs/schema/authenticationinformation-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationinformation-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/authenticationinformation-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-8f1fd41e45bac515231c91a895c84f96-1879d565)
[docs/schema/authenticationtype-element-authenticationinformation-complextypesps15xsdsearchse.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationtype-element-authenticationinformation-complextypesps15xsdsearchse.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/authenticationtype-element-authenticationinformation-complextypesps15xsdsearchse?branch=pr-en-us-8052) | [Details](#user-content-14542ce9dc8f5207b13d361f72e77d9b-1879d565)
[docs/schema/authinfo-element-source-complextypesps15xsdsearchset1.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authinfo-element-source-complextypesps15xsdsearchset1.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/authinfo-element-source-complextypesps15xsdsearchset1?branch=pr-en-us-8052) | [Details](#user-content-5439eb76101423ab4d81b2adc02401a3-1879d565)
[docs/schema/autodeployedwebapplication-element-appprincipaldefinition-complextypesharepoint.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/autodeployedwebapplication-element-appprincipaldefinition-complextypesharepoint.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/autodeployedwebapplication-element-appprincipaldefinition-complextypesharepoint?branch=pr-en-us-8052) | [Details](#user-content-820f5b86c9353d18878646e1c7d9d7b3-1879d565)
[docs/schema/autodeployedwebapplicationdebuginfoappurldefinition-simpletype-sharepoint-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/autodeployedwebapplicationdebuginfoappurldefinition-simpletype-sharepoint-add-in.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/autodeployedwebapplicationdebuginfoappurldefinition-simpletype-sharepoint-add-in?branch=pr-en-us-8052) | [Details](#user-content-206df31f8638a48fd5bc415946baa720-1879d565)
[docs/schema/autodeployedwebapplicationdebuginfodefinition-complextype-sharepoint-add-in-mani.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/autodeployedwebapplicationdebuginfodefinition-complextype-sharepoint-add-in-mani.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/autodeployedwebapplicationdebuginfodefinition-complextype-sharepoint-add-in-mani?branch=pr-en-us-8052) | [Details](#user-content-b398b024876b92d1c15763482c79254f-1879d565)
[docs/schema/baseinfo-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfo-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfo-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-ce20f18c426f331b2b86d076e9db2857-1879d565)
[docs/schema/baseinfo-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfo-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfo-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-e7ed912338752fd988becadc479713a7-1879d565)
[docs/schema/baseinfocollectionofaliasinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofaliasinfotzwwwpjw-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofaliasinfotzwwwpjw-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-a4373ce02c90446030080a42ade3cc09-1879d565)
[docs/schema/baseinfocollectionofaliasinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofaliasinfotzwwwpjw-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofaliasinfotzwwwpjw-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-26b6d13a230027a8874f1b35155f4c74-1879d565)
[docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-18f704b219bc1015b48478fbbe76724d-1879d565)
[docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-858b847baa2fbac78d23794b75097287-1879d565)
[docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-0e7c74139ef7e21b0f7b9b05d6bfa784-1879d565)
[docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-420c9fcf0f63f8c990213de0c63d74d9-1879d565)
[docs/schema/baseinfocollectionofmappinginfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmappinginfotzwwwpjw-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofmappinginfotzwwwpjw-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-fda43918f644f48da43de5398a3b7a0b-1879d565)
[docs/schema/baseinfocollectionofmappinginfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmappinginfotzwwwpjw-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofmappinginfotzwwwpjw-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-ccff20a8015a94d0ee6ab9ba423f5e29-1879d565)
[docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-complextype-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofoverrideinfotzwwwpjw-complextype-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-f406c2700ff32158a954854b39cf2104-1879d565)
[docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-element-sps15xsdsearchset2.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/baseinfocollectionofoverrideinfotzwwwpjw-element-sps15xsdsearchset2?branch=pr-en-us-8052) | [Details](#user-content-7a0f7a798aa4801a0e0e0da8ee6f4b4f-1879d565)

<a id="16aae9269872ae047255b5dfff7b8f96-1879d565"></a>
### [docs/schema/attachment-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/attachment-element-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e2ed7cc00fce0c30c3462cfc51610d5b-1879d565"></a>
### [docs/schema/attachments-element-deploymentmanifest.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/attachments-element-deploymentmanifest.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="72ae00230304cbcd5a01b48a755c455f-1879d565"></a>
### [docs/schema/authenticationdata-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationdata-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="43fd1ca35e429d8b28c571e58cf5c91d-1879d565"></a>
### [docs/schema/authenticationdata-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationdata-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="75903820bea060aa52b8ce7cf3f6a326-1879d565"></a>
### [docs/schema/authenticationinformation-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationinformation-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="8f1fd41e45bac515231c91a895c84f96-1879d565"></a>
### [docs/schema/authenticationinformation-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationinformation-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="14542ce9dc8f5207b13d361f72e77d9b-1879d565"></a>
### [docs/schema/authenticationtype-element-authenticationinformation-complextypesps15xsdsearchse.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authenticationtype-element-authenticationinformation-complextypesps15xsdsearchse.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="5439eb76101423ab4d81b2adc02401a3-1879d565"></a>
### [docs/schema/authinfo-element-source-complextypesps15xsdsearchset1.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/authinfo-element-source-complextypesps15xsdsearchset1.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="820f5b86c9353d18878646e1c7d9d7b3-1879d565"></a>
### [docs/schema/autodeployedwebapplication-element-appprincipaldefinition-complextypesharepoint.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/autodeployedwebapplication-element-appprincipaldefinition-complextypesharepoint.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="206df31f8638a48fd5bc415946baa720-1879d565"></a>
### [docs/schema/autodeployedwebapplicationdebuginfoappurldefinition-simpletype-sharepoint-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/autodeployedwebapplicationdebuginfoappurldefinition-simpletype-sharepoint-add-in.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="b398b024876b92d1c15763482c79254f-1879d565"></a>
### [docs/schema/autodeployedwebapplicationdebuginfodefinition-complextype-sharepoint-add-in-mani.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/autodeployedwebapplicationdebuginfodefinition-complextype-sharepoint-add-in-mani.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ce20f18c426f331b2b86d076e9db2857-1879d565"></a>
### [docs/schema/baseinfo-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfo-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="e7ed912338752fd988becadc479713a7-1879d565"></a>
### [docs/schema/baseinfo-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfo-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="a4373ce02c90446030080a42ade3cc09-1879d565"></a>
### [docs/schema/baseinfocollectionofaliasinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofaliasinfotzwwwpjw-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="26b6d13a230027a8874f1b35155f4c74-1879d565"></a>
### [docs/schema/baseinfocollectionofaliasinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofaliasinfotzwwwpjw-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="18f704b219bc1015b48478fbbe76724d-1879d565"></a>
### [docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="858b847baa2fbac78d23794b75097287-1879d565"></a>
### [docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofcrawledpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="0e7c74139ef7e21b0f7b9b05d6bfa784-1879d565"></a>
### [docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="420c9fcf0f63f8c990213de0c63d74d9-1879d565"></a>
### [docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmanagedpropertyinfotzwwwpjw-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="fda43918f644f48da43de5398a3b7a0b-1879d565"></a>
### [docs/schema/baseinfocollectionofmappinginfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmappinginfotzwwwpjw-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="ccff20a8015a94d0ee6ab9ba423f5e29-1879d565"></a>
### [docs/schema/baseinfocollectionofmappinginfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofmappinginfotzwwwpjw-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="f406c2700ff32158a954854b39cf2104-1879d565"></a>
### [docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-complextype-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-complextype-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="7a0f7a798aa4801a0e0e0da8ee6f4b4f-1879d565"></a>
### [docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-element-sps15xsdsearchset2.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/baseinfocollectionofoverrideinfotzwwwpjw-element-sps15xsdsearchset2.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206071957115208-8052/BuildReport?accessString=a6460d846b0a0e67a727888971126cddabd4b24d388d8197e24fd3938e8da067).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/schema/authenticationdata-element-sps15xsdsearchset2.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: 8ca8cbf0-9123-8716-4de7-d13014e06f7a
-description: "Last modified: March 09, 2015"
+description: AuthenticationData element (SPS15XSDSearchSet2) is an XML element.
```
Comment (RedZephyr13) [inline docs/schema/authenticationdata-element-sps15xsdsearchset2.md:10]: For these articles that don't have much information in the intro paragraph, you can also use this format instead: "Learn about the AuthenticationData element (SPS15XSDSearchSet2)."

**Comment by mSarts** [comment]
**Global effort to fix build validation errors**

@VesaJuvonen 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by andrewconnell — CHANGES_REQUESTED** [review]
When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.

**Comment by mSarts** [comment]
@andrewconnell ms.date updated.

**Comment by mSarts** [comment]
**Global effort to fix build validation errors**

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**